src/app/frontend/assets/images/pin-crossed.svg

```diff
@@ -0,0 +1,20 @@
+<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+  <path d="M7 2h10v2l-2 1v5l3 3v3h-5v4l-1 3-1-3v-4H6v-3l3-3V5L7 4z" fill-rule="evenodd"/>
+  <svg class="kd-cross-style"
+       width="24px"
+       height="24px"
+       viewBox="0 0 24 24">
+    <line stroke="black"
```

Comment:
Please remove the black stroke. It looks strange.

Reply:
<img width="308" alt="Zrzut ekranu 2019-08-26 o 12 24 51" src="https://user-images.githubusercontent.com/2823399/63684193-9d85fa00-c7fc-11e9-980c-c80fe5e598a8.png">

Reply:
Done.